### PR TITLE
Bump memory for test forks to 2GB

### PR DESCRIPTION
**Pull request by tgodzik**
It seems 1GB for a fork was not enough on MacOS. And since we do not have multiple forks running it should be fine.
Current xmx for each process:
- bloop 2GB
- sbt 1GB
- test fork 2GB

With max 7GB it leaves 2GB spare for class space, which should be enough.

### CODE REVIEW

**Review by kpbochenek — APPROVED** [review]

**Review by ckipp01 — APPROVED** [review]

File: tests/unit/src/main/scala/tests/TestingServer.scala [hunk 1]
```diff
@@ -1339,7 +1339,16 @@ final class TestingServer(
           PrettyPrintTree(
             label(uri),
             parent.nodes
-              .map(n => PrettyPrintTree(label(n.nodeUri)))
+              .map { n =>
```
Comment (tgodzik) [inline tests/unit/src/main/scala/tests/TestingServer.scala:1342]: One of the tests for TreeViewLSP suite is failing on Windows. Added something to debug the issue, hopefully it helps.

File: tests/unit/src/main/scala/tests/TestingServer.scala [hunk 2]
```diff
@@ -1339,7 +1339,16 @@ final class TestingServer(
           PrettyPrintTree(
             label(uri),
```
Comment (kpbochenek) [inline tests/unit/src/main/scala/tests/TestingServer.scala:1340]: it fails actually here, and it failed again :/

Comment (tgodzik) [reply tests/unit/src/main/scala/tests/TestingServer.scala]: Right! Changed it. It's super weird it fails, labels are produced for exactly the same keys, which is weird this happens.

Comment (tgodzik) [reply tests/unit/src/main/scala/tests/TestingServer.scala]: So it seems the label exists, but it's still not being found in the map :/

Comment (kpbochenek) [reply tests/unit/src/main/scala/tests/TestingServer.scala]: it is in the map according to logs!!! WTF

Comment (tgodzik) [reply tests/unit/src/main/scala/tests/TestingServer.scala]: Trying with stripping BOM

Comment (kpbochenek) [reply tests/unit/src/main/scala/tests/TestingServer.scala]: nope 😭

Comment (tgodzik) [reply tests/unit/src/main/scala/tests/TestingServer.scala]: Sooo it seems again, it was `c:` instead of `C:` - why Windows? Why!?

**Pull request merged**